tools/devtool

```diff
@@ -1507,10 +1518,46 @@ cmd_release() {
             post_release "$version" "$owner" "$repo" "$token" "$prerelease" "$release_text" ||
             delete_remote_tag "$version" "$remote" "Could not post release to GitHub."
             say "Draft release created successful. Go to https://github.com/$owner/$repo/releases to check it out!"
+
+            say "Updating \`latest\` release version tag..."
+            update_latest_tag "$version" "$remote" ||
+            delete_remote_tag "$version" "$remote" "Could not update latest tag."
     fi
 fi
 }
 
+# `Latest` tag is meant to be used for github latest tagging purposes.
+# This tag is always pointing to the largest version released so far.
+update_latest_tag() {
+    version="$1"
+    remote="$2"
+
+    # List all tags, ignoring the ones that contain `wip` string and fetch
+    # the one with the greatest version.
+    greatest_release=$(git tag | grep -v "wip" | sort -V | tail -1)
```

Comment:
This expression is not robust enough to cover future changes where we might have other non-server tags added. Try matching the version tags in a more exact manner:
```suggestion
    greatest_release=$(git tag | grep -E '^v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+' | sort -V | tail -1)
```